internal/api/health_check.go

```diff
@@ -0,0 +1,38 @@
+package api
+
+import (
+	"net/http"
+
+	"github.com/gorilla/mux"
+)
+
+var buildTag = ""
+var buildHash = ""
+
+type healthCheckResponse struct {
+	Status      string
+	Version     string
+	ReleaseID   string
+	Notes       []string
```

Comment:
This contains the sha1 of the last commit. I didn't found a more elegant solution to add this information to the response.

Reply:
spoke about it offline, suggested to use `Details` instead of `Notes`